contrib/automation_tests/orbit_capture_compatibility.py

```diff
@@ -0,0 +1,64 @@
+"""
+Copyright (c) 2022 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.capture_window import Capture
+from test_cases.connection_window import ConnectToStadiaInstance, FilterAndSelectFirstProcess, LoadCapture
+from test_cases.live_tab import VerifyOneFunctionWasHit
+from test_cases.main_window import DismissDialog, RenameMoveCaptureFile
+from test_cases.symbols_tab import WaitForLoadingSymbolsAndCheckModule
+"""Verify Capture compatibility
+
+This script contains functionality to record a capture with one (new) version of Orbit and then
+test that that this capture can be loaded with another (old) version of Orbit. This is used to test
+that a capture file produces by a (new) Orbit can be read by another (old) version of Orbit.
+
+This needs to be called with one of two values, `record_capture` or `load_capture` and a path to
+where the capture will be saved / from where it will be loaded.
+
+Before the record part of this script is run there needs to be a gamelet reserved and
+"hello_ggp_standalone" has to be started. Further, (new) Orbit needs to be started.
+
+For the load part of this script, (old) Orbit needs to be started.
+
+The script requires absl and pywinauto. Since pywinauto requires the bitness of
+the python installation to match the bitness of the program under test, it needs
+to be run from 64 bit python.
+"""
+
+
+def main(argv):
+    if len(argv) < 3:
+        raise RuntimeError(
+            "Missing argument: You need to pass 2 arguments. The first is `record_capture` or `load_capture` and the second a path to save/load the capture file"
```

Comment:
Nit: this line is too long for the formatting consideration

Reply:
you can use `yapf -i orbit_capture_compatibility.py`

Reply:
did it by hand